networking/ipv6.md

```diff
@@ -1,145 +1,134 @@
-- Each host must have a default IPv6 route.
+- **IPAM**, a CNI plugin configuration with `assign_ipv6` and `assign_ipv4` flags
+- **IPPool**
 
-#### Kubernetes components prerequisites
+### Before you begin
 
-Kubernetes components must be configured to operate with IPv6.
-To enable IPv6, set the following flags.
+**{{site.prodname}} requirements** 
 
-##### kube-apiserver
+  {{site.prodname}} IPAM   
+  The defaut mode is **IPv4**.
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--bind-address` or `--insecure-bind-address` | Should be set to the appropriate IPv6 address or `::` for all IPv6 addresses on the host. |
-| `--advertise-address` | Should be set to the IPv6 address that nodes should use to access the kube-apiserver. |
-| `--service-cluster-ip-range` | Should be set to an IPv6 CIDR that will be used for the Service IPs, the DNS service address must be in this range. |
+**Kubernetes version requirements**
+  - For dual stack (alpha level), 1.16 and 1.17 
+  - 1.15 and earlier supports only one IP stack at a time (IPv6 or IPv4) 
 
-##### kube-controller-manager
+**Kubernetes IPv6 host requirements**
+  - An IPv6 address that is reachable from the other hosts
+  - The sysctl setting, `net.ipv6.conf.all.forwarding`, is set to `1`.    
+    This ensures both Kubernetes service traffic and {{site.prodname}} traffic is forwarded appropriately.
+  - A default IPv6 route
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--master` | Should be set with the IPv6 address where the kube-apiserver can be accessed. |
-| `--cluster-cidr` | Should be set to match the {{site.prodname}} IPv6 IPPool. |
+### How to
 
-##### kube-scheduler
+>**Note**: The following tasks work only for new clusters; you cannot migrate existing IPv4 clusters to IPv6.
+{: .alert .alert-info}
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--master` | Should be set with the IPv6 address where the kube-apiserver can be accessed. |
+**Manifest install**
+- [Enable dual stack, Kubernetes](#enable-dual-stack-kubernetes)
+- [Enable IPv6-only, initial install, Kubernetes](#enable-ipv6-only-initial-install-kubernetes)
+- [Enable IPv6-only, after install, Kubernetes](#enable-ipv6-only-after-install-kubernetes)
 
-##### kubelet
+**OpenShift or Quickstart operator install**
+- [Enable dual stack, OpenShift](#enable-dual-stack-openshift)
+- [Enable IPv6-only, OpenShift](#enable-ipv6-only-openshift)
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--address` | Should be set to the appropriate IPv6 address or `::` for all IPv6 addresses. |
-| `--cluster-dns` | Should be set to the IPv6 address that will be used for the service DNS, this must be in the range used for `--service-cluster-ip-range`. |
-| `--node-ip` | Should be set to the IPv6 address of the node. |
+**Optional**
+- [Configure the Kubernetes control plane to operate over IPv6](#configure-the-kubernetes-control-plane-to-operate-over-ipv6)
```

Comment:
Didn't we agree to move this to a different page?

Reply:
I thought it worked here as an optional step. Is it just not relevant to most users?